avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_10/Array_of_TestRecord_SpecificDeserializer_6709200327098145888_6709200327098145888.java

```diff
@@ -400,7 +399,7 @@ public Array_of_TestRecord_SpecificDeserializer_6709200327098145888_670920032709
                 chunkLen6 = (decoder.mapNext());
             } while (chunkLen6 > 0);
         } else {
-            recordsMap0 = Collections.emptyMap();
+            recordsMap0 = new HashMap<Utf8, com.linkedin.avro.fastserde.generated.avro.SubRecord>(0);
```

Comment:
can these be made Map<CharSequence, ...> ? more general than Utf8, less issue with classes generated to use a different value class for strings?

Reply:
I think it is fine to keep it this way since the key class is tightly coupled with the schema, and if there is string props, the implementation will be `utf8`.